goodfood/complain/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.0.4 on 2018-04-18 16:43
```

Comment:
Delete all the migrations, and do `makemigrations` again we need only one migrations file for a model.